.codeclimate.yml

```diff
@@ -1,5 +1,12 @@
 version: "2"
-engines:
+checks:
```

Comment:
These thresholds come from our developer discussions and are set in our default boilerplate module: https://github.com/SU-SWS/stanford_module_example/blob/8.x-2.x/.codeclimate.yml#L2